CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## [v1.1.20](https://github.com/skyway/skyway-js-sdk/releases/tag/v1.1.20) - 2019-**-**
+
+### Fixed
+
+- Fix bug about support for sdpSemantics options
```

Comment:
It is difficult to understand the fact that `sdpSemantics` is always `plan-b` from this sentence, isnt it?

Reply:
Thank you, I fixed it.